packages/siop-oid4vp/lib/types/SIOP.types.ts

```diff
@@ -176,6 +181,7 @@ export interface AuthorizationResponsePayload {
     | W3CVerifiablePresentation
     | CompactSdJwtVc
     | MdocOid4vpMdocVpToken
+    | EcodedDcqlQueryVpToken
```

Comment:
I don't think the type of vp_token changes with dcql, the format is still the same, so not sure if this is needed (it doesn't matter because string is already present, but i also don't think it's needed)